cargo-shuttle/tests/integration/run.rs

```diff
@@ -5,10 +5,28 @@ use std::{fs::canonicalize, process::exit, time::Duration};
 use tokio::time::sleep;
 
 /// creates a `cargo-shuttle` run instance with some reasonable defaults set.
-async fn cargo_shuttle_run(working_directory: &str) -> u16 {
+async fn cargo_shuttle_run(working_directory: &str, router_ip: bool) -> String {
     let working_directory = canonicalize(working_directory).unwrap();
+
     let port = pick_unused_port().unwrap();
-    let run_args = RunArgs { port };
+
+    let url = if !router_ip {
+        format!("http://localhost:{port}")
+    } else {
+        format!("http://0.0.0.0:{port}")
+    };
+
+    let run_args = if !router_ip {
+        RunArgs {
+            port,
+            router_ip: false,
+        }
+    } else {
+        RunArgs {
+            port,
+            router_ip: true,
+        }
+    };
```

Comment:
```suggestion
    let run_args = RunArgs {
        port,
        router_ip,
    };
```

Just something small

Reply:
Amended on new commit